src/newbank/server/userDB.java

```diff
@@ -5,49 +5,49 @@
 
 import java.util.HashMap;
 
-public class customerDB {
+public class userDB {
```

Comment:
Nice work generalizing this. 
 Preferred to use Title case for class

Reply:
Agree.